sabre_smart_permission_examples/README.md

```diff
@@ -0,0 +1,22 @@
+# Smart Permission Examples
+
+Serveral examples of smart permissions using the sawtooth sabre sdk.
+
+[sabre documentation](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/)
+
+# Smart Permission Boilerplate
+
+```
+extern crate wasm_sdk;
+
+use wasm_sdk::{WasmPtr, WasmPtrList, execute_entrypoint, WasmSdkError, Request};
```

Comment:
wasm_sdk -> sabre-sdk 

Also execute_entrypoint is not execute_smart_permission_entrypoint

Reply:
updated